site/docs/2.2.1/running-on-mesos.html

```diff
@@ -198,7 +198,7 @@ <h2 id="from-source">From Source</h2>
 <ol>
   <li>Download a Mesos release from a
 <a href="http://www.apache.org/dyn/closer.lua/mesos/1.0.0/">mirror</a></li>
-  <li>Follow the Mesos <a href="http://mesos.apache.org/gettingstarted">Getting Started</a> page for compiling and
+  <li>Follow the Mesos <a href="http://mesos.apache.org/getting-started">Getting Started</a> page for compiling and
```

Comment:
I still wouldn't change this as it makes the docs inconsistent with the released artifact, even though it did contain a bug. We'd probably only patch this in case of a serious error.